CHANGELOG.md

```diff
@@ -4,6 +4,10 @@
 
 ##### Enhancements
 
+* Add support for specifying :source with a pod dependency
```

Comment:
Just a reminder, there should be a period and two additional spaces ('.  ') in order to make it start a new line in markdown. Same goes for the changelog in your other PR.